lib/chef/knife/config_list_profiles.rb

```diff
@@ -22,8 +22,11 @@ class Knife
     class ConfigListProfiles < Knife
       banner "knife config list-profiles (options)"
 
+      TABLE_HEADER = [" Profile", "Client", "Key", "Server"].freeze
```

Comment:
Is that extra space in Profile just to left pad it?

Reply:
Yes, in order to align with `*`(currently selected profile). Appended space here. Erlier additional empty column added to achieve the same.

```
table_data = [["", "Profile", "Client", "Key", "Server"]] + profiles.map do |profile|
               [profile[:active] ? "*" : ""] + profile.values_at(:profile, :client_name, :client_key, :server_url).map(&:to_s)
            end
```

Ref: https://github.com/chef/chef/pull/10341/files#diff-060d28bbaa8d398a1e017f21824f3645L99-L101